WcaOnRails/app/models/user_group.rb

```diff
@@ -44,6 +44,10 @@ def roles
     end
   end
 
+  def users
```

Comment:
For future reference, when the migration is complete: (Not relevant to merging this PR!)

I think this can be done even smarter in terms of SQL optimization. When we fully rely on the Groups/Roles model, there will be a `has_many :roles` statement for the Rails database binding.
Then, assuming that the `UserRole` model has a `belongs_to :user` attribute, you can use it here by stating `has_many :users, through: :roles`.

Doing it this way will help you tremendously to optimize DB performance later on.